### ISSUE TITLE
calendar with specific timezone

### ISSUE THREAD

**Issue by SkinnyA** [twinssbc/Ionic2-Calendar#515]
Hello @twinssbc 
I would like to use in the calendar an specific timezone (Madrid, Spain) because even if you are in Dubai or you are in NY, i want to show the date as CET/GMT+1 always.

**Comment by twinssbc** [comment]
@SkinnyA The calendar display the events in local time zone. That is the usual case, for example, if you want to book an event at 4PM CET/UTC+1, you want people in NY see that event at 11 AM UTC-4, so that both place can join the event at the same time. But if your case is really to always display at the same time no matter in which timezone, you have to adjust it at your side. For example, when your code is querying the event, it can pass the timezone information, then the backend code can use that timezone information to adjust the time and return it back.

**Comment by SkinnyA** [comment]
@twinssbc When i load the calendar, it asks for an startTime and endTime date parse as UTC format to load all the events properly. The problem comes here, because i want to load the calendar events without timezone and if i try to load the startTime and endTime without parseing correctly (with UTC format) it throws me an error.

This is my code:
` 
//Here i get all the data from my json
getData() {
    this.http.get('http://localhost:3000/Pedidos').subscribe((resp: any) => {

      resp.forEach((entrada) => {

        let events;
        events = {
          title: entrada.Asunto,
          startTime: entrada.FechaInicio,
          endTime: entrada.FechaFin,
          allDay: false,
          tipo: entrada.Tipo,
        }
//If one type of events is equal to "Aerial" i push the event
        if (events.tipo === "Aereo") {
          events = {
            title: '✈️ ' + entrada.Asunto,
            startTime: new Date(Date.parse(entrada.FechaInicio)),
            endTime: new Date(Date.parse(entrada.FechaFin)),
            allDay: false,
            salidaIATA: entrada.Detalles.SalidaIATA,
            ubicacion: entrada.Ubicacion,
            localizador: entrada.Detalles.Localizador,
            nVuelo: entrada.Detalles.NVuelo,
            aerolinea: entrada.Detalles.Aerolinea,
            salidaNombreAeropuerto: entrada.Detalles.SalidaNombreAeropuerto,
            llegadaNombreAeropuerto: entrada.Detalles.LlegadaNombreAeropuerto,
            duracionHoras: entrada.Detalles.DuracionHoras,
            tipo: entrada.Tipo,

          }

        this.eventSource.push(events);
        this.myCal.loadEvents();
        console.log(this.eventSource);


        }
`

So, if here I try to parse it without timezone, the calendar throws me an error because i must load an event with an startTime and endTime parsed with UTC format.

 startTime: new Date(Date.parse(entrada.FechaInicio)),
 endTime: new Date(Date.parse(entrada.FechaFin)),

So basically i want to load an event with startTime and endTime without timezone because i want to see all my events with no timezone.

I hope I expressed myself well and what im trying to do
Thank you

**Comment by twinssbc** [comment]
@SkinnyA Yes, timezone is just part of a time. Although normally when we tell others about a time without telling timezone, it's because we are in the same place. But if you want to tell people in other place of the world, you need to tell the timezone, otherwise it will cause confusion.
If your date string doesn't contain the timezone, then Date.parse is probably not an option.
There are many ways to construct a Date.
```
new Date()
new Date(value)
new Date(dateString)
new Date(year, monthIndex [, day [, hours [, minutes [, seconds [, milliseconds]]]]])
```
You can write your own string parse logic to get the year, month, day and pass them in the constructor.
Or I recommend you always store numeric value (epoch time) of a date time in your backend.

**Comment by SkinnyA** [comment]
@twinssbc My main problem is about how to load the date because if i try to load it without timezone it throws me an error
This is how i recieve the date 

```
"FechaInicio": "2020-03-14T07:05+02:00",
"FechaFin": "2020-03-14T08:10+02:00"
```

And then i parse it like this 

```
startTime: new Date(Date.parse(entrada.FechaInicio)),
endTime: new Date(Date.parse(entrada.FechaFin)),
```

I parse the date like that because its the only way i can load events in the calendar (with a Date.parse)

However, tomorrow i will try the epoch time and tell you if it work or not.

Thank you for your time and attention,
We are in touch

Ps. Maybe the problem is the hour from the phone? I want to make a mobile app about a fly agency but its so important that the app doesnt takes timezone. 
Even if in the backend theres no timezone, if the mobile is in NY it will automatically parse the time to NY, isnt it? 
How can i avoid or prevent the mobile phone to automatically parse the timezone in JS? Is it possible?

Sorry for my bad english and for my bad explanations :/ I hope u can understand everything properly
Thank you

**Comment by twinssbc** [comment]
@SkinnyA Actually below method call works.
new Date(Date.parse("2020-03-14T07:05"))
You could just remove the timezone part from your date string.
Another way, is you can use
new Date(2020, 2, 14, 7, 5)

**Comment by SkinnyA** [comment]
@twinssbc Ok, i will try it today!
